### PR TITLE
Fixes n tests

### PR DESCRIPTION
**Pull request by gvonkoss**

### CODE REVIEW

File: tests/presenters/package-teaser-presenter.spec.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+const expect = require('chai').expect;
+const Presenter = require('../../src/presenters/package-teaser-presenter');
+const packageFixture = require('../fixtures/package-fixture');
+
+describe.only('Package Teaser Presenter', () => {
```
Comment (TheoLeanse) [inline tests/presenters/package-teaser-presenter.spec.js]: need to remove the `.only` here

Comment (gvonkoss) [inline tests/presenters/package-teaser-presenter.spec.js]: 🙈

File: tests/presenters/package-teaser-presenter.spec.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+		it('returns the three items in package list', () => {
+			subject = new Presenter(packageFixture);
+			expect(subject.packageContent.length).to.equal(3);
+		});
+
+		it('returns the title of items in package list', () => {
+			subject = new Presenter(packageFixture);
+			subject.packageContent.map(content => {
+				expect(content.title).to.equal(packageFixture.contains[packageFixture.contains.indexOf(content)].title);
```
Comment (TheoLeanse) [inline tests/presenters/package-teaser-presenter.spec.js]: not sure but maybe something like `packageFixture.contains.find(blah => blah === content).title` is more readable here?

**Comment by TheoLeanse** [comment]
👍

File: tests/presenters/package-teaser-presenter.spec.js (hunk 2 above)
Comment (TheoLeanse) [inline tests/presenters/package-teaser-presenter.spec.js]: (basically just trying to prove that I did the 👀  here)

**Pull request merged**